### ISSUE TITLE
Remove / relocate “LAN” section on IX page?

### ISSUE THREAD

**Issue by job** [peeringdb/peeringdb#746]
There is a section about MTU + DOT1Q in the IX visual display, which is generally speaking not the most interesting.

It might make sense to move this “LAN” section lower

**Comment by mcmanuss8** [comment]
Seems like putting it at the very bottom of the page on the left makes the most sense

**Comment by arnoldnipper** [comment]
-1 let's keep it where it is. The example @job showed today was not the best one. There is more information in it. And putting it at the very bottom makes it invisible if you have a long list of facilities.

![20200612 PeeringDB 00](https://user-images.githubusercontent.com/16888757/84549932-896aa500-ad09-11ea-9806-0b1ab45b65fd.png)

**Comment by mcmanuss8** [comment]
What if we moved local facilities to the right (below peers)? I think prefixes is more important than dot1q and mtu info.

Does the IX-F info make sense under LAN or should we move that up? I feel like maybe it was put there originally because an IX could have >1 LAN, but now that it doesn't, maybe the IX-F info should be below Notes or its own section.

**Comment by arnoldnipper** [comment]
With removing `ixlan` in v3 entirely we can already start to think where the information should go.

A +1 for moving Facilities (still named local facilities) to the right

**Comment by as44980** [comment]
> Does the IX-F info make sense under LAN or should we move that up? I feel like maybe it was put there originally because an IX could have >1 LAN, but now that it doesn't, maybe the IX-F info should be below Notes or its own section.

+1 to moving anything useful that's historically shown in the LAN section up to the main section for the IX, perhaps between Protocols supported and Notes since the later could  be of variable number of lines?

-1 to moving Facilities to the right as this would place two potentiallty large lists on the right hand side which require scroll bars, and would significantly limit the data which can be shown if the bottom of the Facilities table is aligned with the bottom of the prefixes table on the left...

Unless we mean moving to the right into its own column, perhaps allowing the page to take advantage of more than 50% of the width of monitor manufactured this century.. ;)

(feel free to suggest raising this as a new issue if anyone things its worth considering)

**Comment by arnoldnipper** [comment]
> -1 to moving Facilities to the right as this would place two potentiallty large lists on the right hand side which require scroll bars

For "Public Peering Exchange Points" and "Facilities" we already have scrollbars.  As you can see from the screenshot attached "Facilities" would even come up a bit higher as it is today when being moved to the right. So, where is your point?

![20200618 PeeringDB 00](https://user-images.githubusercontent.com/16888757/84957584-cebd1700-b0fb-11ea-8972-9f581f868cdb.png)

**Comment by arnoldnipper** [comment]
**Summary**
 * Move "Local Facilities" to the right-hand side of the IX page
 * Move "Prefixes" directly under "Contact Information"
 * IX-F information
   * **Either** move IX-F related fields/button into the "Prefixes" box
   * **Or** create a new box which is directly placed under "Prefixes"

**Comment by as44980** [comment]
> > -1 to moving Facilities to the right as this would place two potentiallty large lists on the right hand side which require scroll bars
> 
> For "Public Peering Exchange Points" and "Facilities" we already have scrollbars. As you can see from the screenshot attached "Facilities" would even come up a bit higher as it is today when being moved to the right. So, where is your point?


I agree both have scroll bars so should not cause the 'page' to be larger than can be displayed without scroll bar in the browser; however my concern is that both of the sections which can potentially have enough data to require scroll bar themselves..

Essentially the page becomes lopsided with the majority of the data which I think users will be interested in being on the right hand side of the page..

Whereas perhaps moving one of the static sections such as "Contact Information", "LAN" or "Prefixes" to the right hand side above the "Peers at this Exchange Point" would result in a more balanced layout :)
